### PR TITLE
feat: support multiple ports

### PR DESCRIPTION
**Pull request by sweetpotatoman**
feat: #404

### CODE REVIEW

File: charts/apisix/values.yaml [hunk 1]
```diff
@@ -233,6 +233,12 @@ gateway:
     enabled: true
     servicePort: 80
     containerPort: 9080
+    appendContainerPort: []
```
Comment (tokers) [inline charts/apisix/values.yaml:236]: What about: `additionalContainerPorts`?

Comment (sweetpotatoman) [reply charts/apisix/values.yaml]: Good field name!

File: charts/apisix/templates/configmap.yaml [hunk 2]
```diff
@@ -46,7 +46,11 @@ data:
     {{- else }}
     apisix:    # universal configurations
       {{- if not (eq .Values.deployment.role "control_plane") }}
-      node_listen: {{ .Values.gateway.http.containerPort }}    # APISIX listening port
+      node_listen:    # APISIX listening port
```
Comment (tokers) [inline charts/apisix/templates/configmap.yaml:49]: Could we also support multiple HTTPS ports? Not only for HTTP.

Comment (sweetpotatoman) [reply charts/apisix/templates/configmap.yaml:49]: emm.. In order not to affect the previous, i added `additionalContainerPorts`

**Review by tokers — APPROVED** [review]

**Comment by sweetpotatoman** [comment]
@tao
tao brother, need u approve.

**Review by tao12345666333 — APPROVED** [review]
LGTM

File: charts/apisix/templates/service-gateway.yaml [hunk 3]
```diff
@@ -66,6 +72,10 @@ spec:
   {{- end }}
     protocol: TCP
   {{- end }}
+  {{- range .Values.gateway.tls.additionalContainerPorts }}
+  - name: apisix-gateway-tls-{{ .port | toString }}
+    port: {{ .port }}
+    targetPort: {{ .port }}
```
Comment (tao12345666333) [inline charts/apisix/templates/service-gateway.yaml:78]: end

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

**Comment by tao12345666333** [comment]
@sweetpotatoman please fix the lint error. Thanks

**Comment by sweetpotatoman** [comment]
> @sweetpotatoman please fix the lint error. Thanks

fixed.

**Review by sweetpotatoman — COMMENTED** [review]
fixed

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**